src/webserver/details/HttpBase.h

```diff
@@ -278,9 +270,22 @@ class HttpBase : public ConnectionBase
 
         // We're done with the response so delete it
         res_ = nullptr;
-        // Read another request
+
         doRead();
     }
+
+private:
+    http::response<http::string_body>
+    httpResponse(http::status status, std::string content_type, std::string message) const
+    {
+        http::response<http::string_body> res{status, req_.version()};
+        res.set(http::field::server, "clio-server-" + Build::getClioVersionString());
+        res.set(http::field::content_type, content_type);
+        res.keep_alive(req_.keep_alive());
+        res.body() = std::string(message);
```

Comment:
I think we can `std::move(message)` here instead of copying explicitly. Unless body does not have the move assignment implemented.